cmd/kubelet/app/options/options.go

```diff
@@ -394,6 +393,8 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.MarkDeprecated("chaos-chance", "will be removed in a future version.")
 	fs.StringVar(&f.SeccompProfileRoot, "seccomp-profile-root", f.SeccompProfileRoot, "<Warning: Alpha feature> Directory path for seccomp profiles.")
 	fs.MarkDeprecated("seccomp-profile-root", "will be removed in 1.23, in favor of using the `<root-dir>/seccomp` directory")
+	fs.StringVar(&f.BootstrapCheckpointPath, "bootstrap-checkpoint-path", f.BootstrapCheckpointPath, "<Warning: Alpha feature> Path to the directory where the checkpoints are stored")
```

Comment:
this is marked as an alpha feature, as per our deprecation policy https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-flag-or-cli this can be removed immediately as pointed by @timothysc

Reply:
agree... let's not carry deprecated alpha-level functionality

Reply:
Rolling back:

1. https://github.com/kubernetes/kubernetes/pull/50984, the PR introduced the flag and the initial implementation of pod checkpoint.
2. parts of https://github.com/kubernetes/kubernetes/pull/56040 since the original checkpoint ended using the Checkpoint Manager and there were some tests/ became part of the Kubelet's checkpoint implementation.

Reply:
I'll defer to sig-node reviewers on the partial rollback